recipes/openbabel/2.4.1/meta.yaml

```diff
@@ -35,12 +37,15 @@ requirements:
         - zlib
         - libxml2
         - cairo   # [linux]
+        - xorg-libxau
```

Comment:
During testing using Bioconda `./simulate_travis.py`, it fails due to errors like:
```
[May 10 16:20:25] SERR ==============================
[May 10 16:20:25] SERR *** Open Babel Error  in openLib
[May 10 16:20:25] SERR /usr/local/lib/openbabel/2.4.1/daltonformat.so did not load properly.
[May 10 16:20:25] SERR Error: libXrender.so.1: cannot open shared object file: No such file or directory
```
After adding `xorg-libxrender` to the run requirements, I got errors for missing `libXext.so.6` and so on until I added all these.

Also, if you download the present bioconda package https://anaconda.org/bioconda/openbabel/2.4.1/download/linux-64/openbabel-2.4.1-py35_0.tar.bz2 and run e.g. `ldd lib/openbabel/2.4.1/daltonformat.so`, you'll see that all these xorg libraries are already required.